samples/besu/besu-cli/src/commands/asset/issue.ts

```diff
@@ -0,0 +1,70 @@
+import { GluegunCommand } from 'gluegun'
+import { getNetworkConfig, commandHelp } from '../../helper/helper'
+import { getContractInstance } from '../../helper/besu-functions'
+const Web3 = require ("web3")
+
+const command: GluegunCommand = {
+	name: 'issue',
+
+	run: async toolbox => {
+		const {
+ 			print,
+			parameters: { options }
+		} = toolbox
+		if (options.help || options.h) {
+			commandHelp(
+				print,
+				toolbox,
+				`besu-cli asset issue --network=network1 --contract=path/to/contracts/tokenContract.json --account=1 --amount=10`,
+				'besu-cli asset issue --network=<network1|network2> --contract=<path-to-tokencontract-json>--account=<account-index> --amount=<issue-ammount>',
+				[
+					{
+						name: '--network',
+						description:
+							'network for command. <network1|network2>'
+					},
+					{
+						name: '--contract',
```

Comment:
Can we get this from `networkConfig` instead of passing it here? We are setting the contract there also.
Just to avoid this parameter.